File: mentoring/MusicTest/app/src/main/java/com/example/musictestmentoring/MainActivity.java
package com.example.musictestmentoring;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

import com.airbnb.lottie.LottieAnimationView;

public class MainActivity extends AppCompatActivity {

    private static String TAG = MainActivity.class.getSimpleName();

    Button mBtnStart;
    Button mBtnStop;
    LottieAnimationView mLottieIcon;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Log.d(TAG,"onCreate");
     //   EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
    /*    ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        }); */

        mLottieIcon= findViewById(R.id.lottie_icon);

        mBtnStart= findViewById(R.id.btn_start);
        mBtnStart.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startMusicService();
                // start 클릭 시 로티 애니메이션 동작
                mLottieIcon.playAnimation();
            }
        });

        mBtnStop= findViewById(R.id.btn_stop);
        mBtnStop.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                stopMusicService();
                mLottieIcon.pauseAnimation();
            }
        });
    }

    private void startMusicService(){
        Intent intent = new Intent(this, MusicService.class);
        startService(intent);
        Log.d(TAG,"startMusicService");
    }

    private void stopMusicService(){
        Intent intent = new Intent(this, MusicService.class);
        stopService(intent);
        Log.d(TAG,"stopMusicService");
    }

}

File: week5/Project5/app/src/main/java/com/example/project5/MainActivity.kt
package com.example.project5

import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val textView = findViewById<TextView>(R.id.textView)
        val editText = findViewById<EditText>(R.id.editText)
        val button = findViewById<Button>(R.id.button)

        // 버튼 클릭시
        button.setOnClickListener{
            // editText의 내용이 textView의 내용으로 변경
            val str = editText.text.toString()
            textView .text =  str
        }
    }
}